packages/shared/src/utils/contract/contract.ts

```diff
@@ -81,17 +82,12 @@ export const getAccuBarriersDefaultTimeout = (symbol: string) => {
 export const getAccuBarriersDTraderTimeout: TGetAccuBarriersDTraderTimeout = ({
     barriers_update_timestamp,
     has_default_timeout,
-    should_update_contract_barriers,
     tick_update_timestamp,
     underlying,
 }) => {
     if (has_default_timeout || !tick_update_timestamp) return getAccuBarriersDefaultTimeout(underlying);
-    const animation_correction_time =
-        (should_update_contract_barriers
-            ? getAccuBarriersDefaultTimeout(underlying) / -4
-            : getAccuBarriersDefaultTimeout(underlying) / 4) || 0;
     const target_update_time =
-        tick_update_timestamp + getAccuBarriersDefaultTimeout(underlying) + animation_correction_time;
+        tick_update_timestamp + getAccuBarriersDefaultTimeout(underlying) + ANIMATION_CORRECTION_TIME;
```

Comment:
Updating animation correction time because it does not depend on the symbol's speed or the type of API (for a contract or not). It's the time required for a current spot to get visually updated on the chart after we get it from API. It's taken into account in order to calculate the animation timeout correctly and ensure steady pace of barriers animation.